share/wake/lib/system/remote_cache_api.wake

```diff
@@ -499,8 +494,7 @@ export def rscApiGetFileBlob ((CacheSearchBlob _ uri): CacheSearchBlob) (path: S
 def strToBytes (str: String): List Integer =
     str
     | explode
-    # FIXME: unicodeToInteger may return values larger than u8 which will break the serialization
```

Comment:
May still be worth leaving a `FIXME` with updated text related to high-code-point values.  I'm actually surprised emoji worked; I don't actually know how that would have happened unless there's a bug in `explode`.

Reply:
I've marked it in my task list. Its a bit easier to keep track of there